ui/panels-plugin/src/plugins/line-chart/tooltip/SeriesMarker.tsx

```diff
@@ -22,19 +22,16 @@ function SeriesMarker(props: SeriesMarkerProps) {
   return (
     <Box
       sx={{
-        display: 'flex',
-        alignItems: 'center',
-        margin: '0 6px 0 0',
+        borderRadius: '2px',
+        display: 'inline-block',
+        height: '12px',
+        marginRight: 1,
+        marginTop: 0.25,
+        verticalAlign: 'top',
+        width: '12px',
       }}
-    >
-      <Box
-        sx={{
-          width: '10px',
-          height: '10px',
-          backgroundColor: markerColor,
-        }}
-      ></Box>
-    </Box>
+      style={{ backgroundColor: markerColor }}
```

Comment:
I moved the `markerColor` to an inline style to avoid excessive class generation by styled components.

Reply:
Ahh, was that what was causing a new class on the element every time you moved the mouse by like 1px?